src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -38,6 +38,34 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
                 .anyMatch(preppedWord::equalsIgnoreCase);
     }
 
+    /**
+     * Returns true if the {@code sentence} contains the {@code tag}.
+     *   Ignores case, but a full word match is required.
+     *   <br>examples:<pre>
+     *       containsWordIgnoreCase("ABc def", "abc") == true
+     *       containsWordIgnoreCase("ABc def", "DEF") == true
+     *       containsWordIgnoreCase("ABc def", "AB") == false //not a full word match
+     *       </pre>
+     * @param sentence cannot be null
+     * @param tag cannot be null, cannot be empty, must be a single word
+     */
+    public static boolean containsTagIgnoreCase(String sentence, String tag) {
```

Comment:
So far the tag is `String` instead of `Tag` Class?

Reply:
Yes it checks if the person contains any tag name specfic in String